Update readmes to include details on the benchmarking framework (#226)
awslabs/aws-lc: tests/ci/benchmark_framework/README.md

## Overview

> This folder contains various scripts and tools used by the benchmarking framework in order to help it interprete the results obtained by AWS-LC's benchmarking tools.

Reviewer: ```suggestion
This folder contains various scripts and tools used by the benchmarking framework to process results obtained by  the AWS-LC's benchmarking tools.
```

> As such, they can also be used by any person who wants to interpret their own data also obtained from the AWS-LC benchmarking tools.

Reviewer: Reformulate without "person"
